.lintstagedrc

```diff
@@ -1,5 +1,5 @@
 {
 	"*.{png,jpeg,jpg,gif,svg}": ["imagemin-lint-staged", "git add"],
-	"*.js": ["npm run lint:js"],
-	"*.css": ["npm run lint:css"],
+	"*.js": ["eslint"],
```

Comment:
Why did you change these?

Reply:
Because the scripts referenced in the `.js` and `.css` executes linting on the whole scripts and styles folders respectively.
```
"lint:js": "eslint src/scripts/**/*.js --color",
"lint:css": "stylelint src/styles/**/*.css --color --fix",
```
Which I believe is not the intended behavior, we only want to lint the staged files